dev/env/manifests/rhacs-operator/03-operators-config.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: operator-config
+  namespace: "$ACSMS_NAMESPACE"
+data:
+  operator-config.yaml: |-
+    - gitref: 4.0.0
+      image: quay.io/rhacs-eng/stackrox-operator:4.1.1
```

`gitRef` and image tag are different, this would act as a bad example. Can you adjust the values to match each other?

```suggestion
    - gitRef: 4.1.1
      image: quay.io/rhacs-eng/stackrox-operator:4.1.1
```